evmos/chain.json

```
@@ -93,12 +93,20 @@
       {
         "address": "https://tendermint.bd.evmos.org:26657",
         "provider": "evmos.org"
+      },
+      {
+        "address": "https://rpc.evmos.ezstaking.io",
+        "provider": "EZStaking.io"
       }
     ],
     "rest": [
       {
         "address": "https://rest.bd.evmos.org:1317",
         "provider": "evmos.org"
+      },,
```

Just noticed this invalid JSON in my fork
```suggestion
      },
```